### PR TITLE
Add ship client and additional DTOs

### PR DESCRIPTION
**Pull request by argonavis80**
- Add `ShipClient`.
- Add additional DTOs for ship details.
- Add missing enum values in `FeedState`.
- Fixed some typos.

### CODE REVIEW

File: BlueTracker.SDK.Performance/Clients/ShipClient.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        /// </summary>
+        /// <param name="imoNumber">IMO number of the vessel.</param>
+        /// <param name="enableBatchMode">True to enable batch mode; false to disable batch mode.</param>
+        /// <returns>The ship object.</returns>
+        /// <remarks>
+        /// When batch mode is enabled, data items pushed to the vessel do not trigger an immediate re-calculation.
+        /// This is useful, if big amounts of items needs to be pushed. When the batch mode is disabled, a full
+        /// re-calculation is triggered.
+        /// </remarks>
+        public Ship SetBatchMode(int imoNumber, bool enableBatchMode)
```
Comment (thegoodsoldier) [inline BlueTracker.SDK.Performance/Clients/ShipClient.cs:77]: This endpoint is also in the ReportClient. Please decide whether we need it in both clients.

File: BlueTracker.SDK.Performance/Clients/ShipClient.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        /// </summary>
+        /// <param name="imoNumber">IMO number of the ship.</param>
+        /// <returns>The ship object.</returns>
+        /// <remarks>
+        /// Calling this method, will put the ship to <see cref="FeedState.Pending"/> mode, and a re-calulcation
+        /// job is queued. The method call will then return. During execution of the re-calculation job, the ships
+        /// feed state will be set to <see cref="FeedState.Calculation"/>. When the job is done, the ship feed 
+        /// state will be set back to <see cref="FeedState.Online"/>.
+        /// </remarks>
+        public Ship Calculate(int imoNumber)
```
Comment (thegoodsoldier) [inline BlueTracker.SDK.Performance/Clients/ShipClient.cs:94]: Same as above.

File: BlueTracker.SDK.Performance/Clients/ShipClient.cs (hunk 1 above)
Comment (argonavis80) [reply BlueTracker.SDK.Performance/Clients/ShipClient.cs:77]: Done.

File: BlueTracker.SDK.Performance/Clients/ShipClient.cs (hunk 2 above)
Comment (argonavis80) [reply BlueTracker.SDK.Performance/Clients/ShipClient.cs:94]: Done.

**Pull request merged**